multinode_runfabtests.sh

```diff
@@ -77,4 +134,16 @@ if [ ${PROVIDER} == "efa" ]; then
     # So here we run it separately using "-b" option
     echo "Run fi_dgram_pingpong with out-of-band synchronization"
     run_dgram_pingpong_with_b ${SERVER_IP} ${CLIENT_IP}
+
+    # Run fi_rdm_tagged_bw with fork when different environment variables are set.
+
+    echo "Run fi_rdm_tagged_bw with fork"
+    run_rdm_tagged_bw_fork ${SERVER_IP} ${CLIENT_IP}
```

Comment:
With out setting environment variables, this test will actually fail, which is expected. How to account for that?

Reply:
We expect non zero return status when no environment variables is set "https://github.com/awslabs/libfabric-ci-scripts/pull/140/files#diff-effebc9b96458b408492a90d2789d72f1e79cc496af4392c51cc52e595686e6dR75".